### PR TITLE
 Adding Missed Activation Functions for Grouped 2D/3D Convolutions

### PR DESCRIPTION
**Pull request by ThruptiRajLakshmanaGowda**

### CODE REVIEW

File: example/62_convnd_activ/unary/convnd_fwd_xdl_logistic_fp16.cpp [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2023-2024, Advanced Micro Devices, Inc. All rights reserved.
```
Comment (geyyer) [inline example/62_convnd_activ/unary/convnd_fwd_xdl_logistic_fp16.cpp]: Could you update it to 2024 only?

File: example/62_convnd_activ/unary/convnd_fwd_xdl_passthrough_fp16.cpp [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2023-2024, Advanced Micro Devices, Inc. All rights reserved.
```
Comment (geyyer) [inline example/62_convnd_activ/unary/convnd_fwd_xdl_passthrough_fp16.cpp]: Same here

File: example/62_convnd_activ/unary/convnd_fwd_xdl_swish_fp16.cpp [hunk 3]
```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2023-2024, Advanced Micro Devices, Inc. All rights reserved.
```
Comment (geyyer) [inline example/62_convnd_activ/unary/convnd_fwd_xdl_swish_fp16.cpp]: And here

File: include/ck/tensor_operation/gpu/element/unary_element_wise_operation.hpp [hunk 4]
```diff
@@ -961,6 +961,24 @@ struct Elu
...
+    template <typename T>
+    __host__ __device__ void operator()(T& y, const T& x) const
+    {
+        static_assert(is_same<T, float>::value || is_same<T, double>::value ||
+                          is_same<T, half_t>::value || is_same<T, int32_t>::value ||
+                          is_same<T, int8_t>::value,
+                      "Data type is not supported by this operation!");
+        T casted_alpha  = type_convert<T>(alpha_);
+        constexpr T one = type_convert<T>(1);
+        y               = one * casted_alpha / (one + ck::math::exp(-x) * casted_alpha);
```
Comment (geyyer) [inline include/ck/tensor_operation/gpu/element/unary_element_wise_operation.hpp]: Multiplication by one looks hacky, why do we need it here?

**Review by geyyer — COMMENTED** [review]
Good job, Thrupti! Please address the comments and this PR is good to go!

**Review by geyyer — APPROVED** [review]
LGTM!

**Review by zjing14 — APPROVED** [review]

**Pull request merged**